Library/HorizontalBarPlot.Rmd

```diff
@@ -28,33 +31,55 @@ data2 <- read_dta("https://github.com/worldbank/r-econ-visual-library/raw/master
 head(data1)
 head(data2)
 
+# Data Dictionary (data1) ---------------
+  ## Data Source: Data Source: Das, Jishnu, Liana Woskie, Ruma Rajbhandari, Kamran Abbasi, and Ashish Jha. “Rethinking assumptions about delivery of healthcare: implications for universal health coverage.” Bmj 361 (2018).
+  ## Tidy format? Yes
+  ## Unit of Observation: Individual clinics
+    # study: Clinic type identifier
+    # hours: Total daily time seeing patients (hours)
+
+# Data Dictionary (data2) ---------------
+  ## Data Source:
+  ## Tidy format? Yes
+  ## Unit of Observation: Individual clinics
+  ## Variables of Interest
+     # study: Clinic type identifier
+     # totime: Total daily time seeing patients (minutes)
+
+
 # Data wrangling ---------------
 
 # Generate hour variable from minute variable
-data2 <- data2 %>% mutate(hours = tottime / 60)
+data2 <- data2 %>% mutate(hours = tottime / 60) # Convert time variable to hourly format 
 
 # Append two datasets
 data <- bind_rows(data1, data2) # bind_rows append the two given datasets
```

Comment:
edit the data to include this part from the start (remove this wrangling from here)